suricata/update/main.py

```diff
@@ -1279,8 +1284,8 @@ def _main():
     files = None
 
     if not test_suricata(suricata_path):
-        logger.error("Suricata test failed, aborting.")
-        logger.error("Restoring previous rules.")
+        emit("Suricata test failed, aborting.")
+        emit("Restoring previous rules.")
```

Comment:
This should not be a fatal error, even with `--fail` as it will prevent restoring the good rules after a Suricata test failure.